packages/react-scripts/template/src/App.css

```diff
@@ -1,21 +1,28 @@
 .App {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  padding: 3rem;
   text-align: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.App > *+* {
```

Comment:
We can re-write this behavior in many ways or explicitly put margins on each child of the component. I will go ahead and do that!

As a side note I have always found this selector very useful, especially for container components. I think an axiomatic approach to layout complements very nicely the principles of react. Once you get your head around the _magic_ it makes sense. See [lobotomized owls](http://alistapart.com/article/axiomatic-css-and-lobotomized-owls).

> I am a very poor mathematician, but I have a great fondness for Euclid’s postulates: a set of irreducible rules, or axioms, that form the basis for complex and beautiful geometries. Thanks to Euclid, I understand that even the most complex systems must depend on foundational rules, and CSS is no different. Although modularization of a complex interface is a necessary step in its maturation, any interface that does not follow basic governing tenets is going to lack clarity.

> The owl selector allows you to control flow content, but it is also a way of relinquishing control. By styling elements according to context and circumstance, we accept that the structure of content is—and should be—mutable. Instead of prescribing the appearance of individual items, we build systems to anticipate them.